x-pack/plugins/cases/public/components/all_cases/index.test.tsx

```diff
@@ -138,26 +138,6 @@ describe('AllCases', () => {
     });
   });
 
-  it('should not allow the user to enter configuration page with basic license', async () => {
-    useGetActionLicenseMock.mockReturnValue({
-      ...defaultActionLicense,
-      data: {
-        id: '.jira',
-        name: 'Jira',
-        minimumLicenseRequired: 'gold',
-        enabled: true,
-        enabledInConfig: true,
-        enabledInLicense: false,
-      },
-    });
-
-    const result = appMockRender.render(<AllCases />);
-
-    await waitFor(() => {
-      expect(result.getByTestId('configure-case-button')).toBeDisabled();
-    });
-  });
-
   it('should allow the user to enter configuration page with gold license and above', async () => {
```

Comment:
This one might also be outdated 😅 
`'should allow the user to enter configuration page with basic license and above'`?
and then maybe change `minimumLicenseRequired` on line 147?

Reply:
Deleted. It's always enabled now, testing it feels to excessive for something this trivial